lib/hyrax/configuration.rb

```diff
@@ -227,9 +227,13 @@ def curation_concerns
     # @see lib/wings
     # @see https://github.com/samvera-labs/valkyrie
     def valkyrie_metadata_adapter
-      Valkyrie::MetadataAdapter.find(@valkyrie_metadata_adapter || :wings_adapter)
+      Valkyrie::MetadataAdapter.find(@valkyrie_metadata_adapter)
```

Comment:
looks like we still need the default, here.

We could (should?) also get a default into the generator for the config. But either way, we'd need this to support apps that generated their config with an older version,